onnx/test/shape_inference_test.py

```diff
@@ -67,7 +67,7 @@ def _make_graph(
                 )
                 input_value_infos.append(
                     make_tensor_value_info(
-                        "UNKNOWN_SHAPE_" + seed_name, TensorProto.INT64, ()
+                        "UNKNOWN_SHAPE_" + seed_name, TensorProto.INT64, (None,)
```

Comment:
Just a quick note on this. getShapeInput's rank inference [fails hard](https://github.com/onnx/onnx/blob/3dafb54b11a496cd933de17fd303324d0686fc3b/onnx/defs/shape_inference.cc#L475) when the shape is not 1 dimensional which makes this necessary. I would be in favour of the function setting `found` to `false` and returning instead since this lets the caller determine how to proceed.